src/components/Trade/types.ts

```diff
@@ -0,0 +1,42 @@
+import { AssetId } from '@shapeshiftoss/caip'
+import { Trade, TradeQuote } from '@shapeshiftoss/swapper'
+import { Asset, chainAdapters, ChainTypes } from '@shapeshiftoss/types'
+
+export enum TradeAmountInputField {
+  BUY = 'BUY',
+  SELL = 'SELL',
+  FIAT = 'FIAT',
+}
+
+export type TradeAsset = {
+  asset: Asset
+  amount?: string
+}
+
+export type TradeProps = {
+  defaultBuyAssetId?: AssetId
```

Comment:
```suggestion
  defaultBuyAssetId: AssetId
```

shouldn't a default always be defaulted?